src/App.tsx

```diff
@@ -1,61 +1,111 @@
+import { useState } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './types/todo';
+import { getUserById } from './services/userService';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+const initialTodo: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId) || null,
+}));
```

Comment:
Ensure that data preparation, such as mapping todos with user data, is centralized in the `App` component. This aligns with the checklist requirement to prepare data in one place.